src/main/java/bob/Task/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package bob.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+/**
+ * {@inheritDoc}
+ */
+public class Deadline extends Task {
+    private LocalDateTime date;
+
+    public Deadline(String name, LocalDateTime date) {
+        super(name);
+        super.type = "D";
+        super.setStatus(0);
+        this.date = date;
+    }
+
+    @Override
+    public String printStatus() {
+        return "[D] " + Task.statusSymbols[super.getStatus()] + " " + this + " (by: " +
+                date.format(DateTimeFormatter.ofPattern("dd MMM yyy HH:mm")) + ")";
+    }
+
+    @Override
+    public String toString() {
+        return super.getName();
+    }
```

Comment:
Consider writing Javdoc comments for public methods if you have time, same goes for the other classes too

Reply:
Definitely will work on it in the next increments 😅